lib/Conversion/SeqToSV/FirMemLowering.cpp

```diff
@@ -226,7 +227,7 @@ FirMemLowering::createMemoryModule(FirMemConfig &mem,
   auto addCommonPorts = [&](StringRef prefix, size_t idx) {
     addInput(prefix, idx, "_addr", addrType);
     addInput(prefix, idx, "_en", bitType);
-    addInput(prefix, idx, "_clk", bitType);
+    addInput(prefix, idx, "_clk", clkType);
```

Comment:
Should this be part of #6053? There are a few other tweaks to Seq that look like it might belong to another PR.

Reply:
I don't think so - ARC detects memories and it's best if it sees a clock type after this pass instead of some clock conversions.

Reply:
I split off some stuff into this PR: https://github.com/llvm/circt/pull/6062